fml/shared_thread_merger.cc

```diff
@@ -85,6 +86,14 @@ bool SharedThreadMerger::IsMergedUnSafe() const {
   return !IsAllLeaseTermsZeroUnSafe();
 }
 
+bool SharedThreadMerger::IsEnabledUnSafe() const {
+  return enabled_;
+}
+
+void SharedThreadMerger::SetEnabledUnSafe(bool enabled) {
+  enabled_ = enabled;
+}
+
```

Comment:
I'm a bit uncertain about which threads are allowed to call this.

Reply:
OK, I will think about it